test/e2e/validation_test.go

```diff
@@ -53,6 +54,53 @@ type winService struct {
 	description string
 }
 
+// testNodesBecomeReady tests that all Windows nodes become ready and schedulable at least once.
```

Comment:
I don't understand the `at least once` comment. Why is that there?

Reply:
Thought process is that schedulable can be changed by WMCO at will. Removing it for clarity